doc/classes/FuncRef.xml

```diff
@@ -33,7 +34,7 @@
 			<argument index="0" name="instance" type="Object">
 			</argument>
 			<description>
-				Set the object on which to call the referenced function. This object must be of a type actually inheriting from [Object], not a built-in type such as [int], [Vector2] or [Dictionary].
+				Sets the object on which to call the referenced function. This object must be of a type actually inheriting from [Object], not a built-in type such as [int], [Vector2] or [Dictionary].
```

Comment:
Ref writing guidelines: "Don’t repeat the method’s name. It already states it sets":

    The object containing the referenced function.